### PR TITLE
Improve styles for mobile

### PR DESCRIPTION
**Pull request by arathjz**
**What:**
- Improve styles for mobile
- Update footer links

**Why:**
Relates to: https://app.asana.com/0/1159164196409129/1191556884196657/f

**How:**
- Adapt sections to mobile devices
- Set padding y and x for mobile
- Refactor slider to detect slide change on a touch event
- Adapt images for mobile
- Replace inline background gradient to tailwind background gradient

#### Media
https://www.loom.com/share/79d3d982677b44e289528fb6f59484dd

### CODE REVIEW

File: src/components/Footer/index.tsx [hunk 1]
```diff
@@ -42,43 +48,78 @@ const Footer = () => {
...
-              { label: 'Open source', href: '' }
+              {
+                label: 'Contact',
+                href: 'https://tools.debtcollective.org/contact'
+              },
+              {
+                label: 'Donate',
+                href: 'https://membership.debtcollective.org/'
+              }
+              // { label: 'Debt Collective and Open Source Technology', href: '' }
```
Comment (duranmla) [inline src/components/Footer/index.tsx:109]: Do you need this comment here?
Maybe unused code can be removed.

File: src/tailwind.css [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@import url('https://fonts.googleapis.com/css?family=Libre+Franklin:300,400,600');
+@import url('https://fonts.googleapis.com/css?family=Libre+Franklin:300,400,600,700,800,900');
```
Comment (duranmla) [inline src/tailwind.css:1]: Daaamn... do we really need all of those? This add a lot into the first loading time I will assume.

**Review by duranmla — APPROVED** [review]
LGTM. couple comments but nothing blocker!

File: src/components/Footer/index.tsx [hunk 3]
```diff
@@ -3,37 +3,43 @@ import dcLogo from '@static/debt-collective-logo.svg';
...
+    <footer className="bg-gray px-x-screen-spacing py-y-screen-spacing lg:p-desktop-screen-spacing">
       <div className="max-w-8xl mx-auto">
         <NewsLetter />
         <hr className="w-full border-gray-100 mb-16 lg:mb-20" />
         <section className="flex flex-col items-center mb-16 lg:flex-row lg:justify-between">
           <img src={dcLogo} alt="debt collective logo" />
           <ul className="flex mt-10 lg:mt-0">
             <li className="mr-10">
-              <a href="">
-                <img src={facebookIcon} alt="facebook" />
```
Comment (orlando) [inline src/components/Footer/index.tsx:21]: I think we should use font for these icons. We can address this later

**Review by orlando — APPROVED** [review]
LGTM

**Pull request merged**